Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -247,6 +248,10 @@ void LoadEventNexus::init() {
 
   declareProperty(std::make_unique<PropertyWithValue<bool>>("LoadLogs", true, Direction::Input),
                   "Load the Sample/DAS logs from the file (default True).");
+
```

Comment:
Could you help me understand the overlap with this new property and `LoadLogs` above - it's not completely clear to me from the diff.

Reply:
If `LoadLogs` is checked, `LoadNexusLogs` will be called as a child algorithm afterward, and then only load a fixed list of logs items (and process some of them) (https://docs.mantidproject.org/nightly/algorithms/LoadNexusLogs-v1.html?highlight=loadnexuslog#data-loaded-from-nexus-file).
On the contrary, `LoadAllLogs` just recursively go through the nexus and load everything that is not data as it is.
Perhaps `LoadRawLogs` would be clearer though ?

Reply:
Ah I see so it's searching too. Thanks. I'd like to avoid the word Raw as it's an ISIS file format so could get confused with that. Is it worth the parameter having something like Search/Find/Discover in the name? Maybe `SearchForLogs`?

Reply:
I don't think adding search is a good idea, it's more about loading than searching. This is what is done in all ILL loaders: retrieve and load all the metadata. It's striking that LoadLogs option doesn't do that.

Reply:
I think this is a hangover from sample/run information always being in the same place in SNS/ISIS files.

I'd be happy with `LoadAllLogs` but can we perhaps improve the documentation on both `LoadLogs` and `LoadAllLogs` properties to clarify the difference. As you say people may be surprised that `LoadLogs` doesn't load everything and will wonder where it loads from.